t/shdict.t

```diff
@@ -999,3 +999,26 @@ number expected, got string
 --- no_error_log
 [alert]
 [crit]
+
+
+
+=== TEST 31: get stats
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        content_by_lua_block {
+            local dogs = ngx.shared.dogs
+            local total_used, total_size = dogs:stats()
+            ngx.say("total_used type: ", type(total_used))
+            ngx.say("total_size type: ", type(total_size))
+        }
+    }
+--- request
+GET /t
+--- response_body
+total_used type: number
+total_size type: number
+--- no_error_log
+[error]
+[alert]
+[crit]
```

Comment:
I think we need to add more test cases for the following scenarios:

1. the shdict is empty,
2. the shdict is full,
3. the shdict is half-full,
4. the shdict has tiny key-value pairs (just a few bytes).
5. the shidct has big key-value pairs (larger than a page size).

Of course, we all need to check the actual values in all these cases rather than simply check the values are of the number type. Otherwise it makes little sense :)